rust/worker/src/execution/orchestration/hnsw.rs

```diff
@@ -99,7 +100,16 @@ impl HnswQueryOrchestrator {
                 return;
             }
         };
-        let input = PullLogsInput::new(collection_id, 0, 100);
+        let end_timestamp = SystemTime::now().duration_since(UNIX_EPOCH);
+        let end_timestamp = match end_timestamp {
+            // TODO: change protobuf definition to use u64 instead of i64
```

Comment:
created a ticket for myself so we don't forget - https://linear.app/trychroma/issue/CHR-389/change-logs-proto-to-use-u64-instead-of-i64-for-timestamp

Reply:
Thank you!